app/models/ability.rb

```diff
@@ -31,6 +31,10 @@ def not_signed_in
       event.state == 'confirmed'
     end
 
+    can [:show, :events], Schedule do |schedule|
```

Comment:
I still thing that https://github.com/openSUSE/osem/pull/2389/commits/ea312eb728a20770ac9da0132bbc9214840b2e93 was correct... there is not `schedule` method in the [conference controller](https://github.com/openSUSE/osem/blob/master/app/controllers/conferences_controller.rb) :confused:

Reply:
you can also freely invent your own "method" names and check for them.